pnpm-lock.yaml

```diff
@@ -1584,38 +1592,34 @@ packages:
       buffer: 6.0.3
     dev: true
 
-  /@prisma/studio-pcw/0.459.0_839d6f5e1e290969a8d17594ebaffe12:
+  /@prisma/studio-pcw/0.459.0:
     resolution: {integrity: sha512-V3bvY8XXtRq4dRxtAci3+MAHee0joQU1kVmtjT2RIbFXonnOSo3jkqRMLV2OpNx96oysS5h1u3K86a/EavG2bw==}
     engines: {node: '>= 12'}
     peerDependencies:
       '@prisma/client': '*'
       '@prisma/sdk': '*'
     dependencies:
-      '@prisma/client': link:packages/client
-      '@prisma/sdk': link:packages/sdk
```

Comment:
Could you check which version of pnpm you have, post it here and then try upgrading to latest v6 with 
```
pnpm add -g pnpm
```
And update the lockfile again? I think these will be back after. (Theory to be confirmed)

Reply:
Currently I have pnpm@6.32.2

Reply:
`pnpm-lock.yaml` doesn't get updated even when I purge the existing `node_modules` folder.